include/ublksrv_tgt.h

```diff
@@ -112,27 +80,11 @@ struct co_io_job {
 
     co_handle_type coro;
 
-    co_io_job(co_handle_type h): coro(h){}
-
-    void resume() {
-        coro.resume();
-    }
+    co_io_job(co_handle_type h): coro(h) {}
 
     operator co_handle_type() const { return coro; }
 };
 
-/*
- * c++20 is stackless coroutine, and can't handle nested coroutine, so
- * the following two have to be defined as macro
- */
-#define co_io_job_submit_and_wait(tag) do {		\
```

Comment:
Per my experience,  it is very convenient to keep 'tag' as the parameter when debug some tough coroutine issue.

So I'd suggest to keep the 'tag' parameter here, otherwise this PR looks fine.

Reply:
As far as I can tell, this `tag` macro parameter is discarded by the preprocessor in the current code, so I'm not sure how it was helping debug coroutine issues. After applying the preprocessor, here is my change:

Before:

```
do {
    co_await ublk_suspend_always();
} while (0);
```

After:

```
    co_await std::suspend_always();
```

Can you clarify how you'd like me to improve this PR?

Reply:
I meant that debug code can be added inside co_io_job_submit_and_wait(), with 'tag' printed.

As I mentioned, except for this one, other change is fine.